### DISCUSSION TITLE
UI component/widget libraries for Vue 3 and Nuxt 3

### DISCUSSION THREAD

**Discussion by drewda** [nuxt/nuxt#17189] [Questions]
This is such a general question that this may not be the best forum for it, but I will try asking here as others may be in the same situation.

We want to start upgrading our Nuxt 2-based sites and apps to Nuxt 3. However, we're finding that the main UI component/widget library that we use (Buefy + Bulma) will not making the transition to Vue 3.

I totally understand that's a choice to be made by the maintainers of each UI component/widget library. But it leaves us with a very open question of when and how to upgrade to Nuxt 3.

How are other Nuxt 3 users making the transition? What UI component/widget libraries are you using, if any?

**Comment by angelhdzmultimedia** [comment]
Vuetify and Quasar.

**Comment by Kunsih6** [comment]
If you are able to work with [TailwindCSS](https://tailwindcss.com/) you should try [Tailwind UI](https://tailwindui.com/) or [daisyUI](https://daisyui.com/ ). But Vuetify has been my choice to develop dashboards apps.
